client/build

```diff
@@ -51,9 +56,9 @@ if [ $suc -eq 1 ]; then
         ret="${PIPESTATUS[0]}"
         if [ $ret -eq 0 ]; then
             printf "\x1b[1;97m\x1b[92msucceeded\x1b[m\x1b[1;97m atomify build\n\x1b[m"
-            if cargo run -q ../../client/gen/graphics.wat; then
+            if cargo run -q $gen_dir/graphics.wat; then
```

Comment:
disassemble and additional steps could be moved in its own make target and composed using dependencies...

Reply:
Script is dropped entirely